### ISSUE TITLE
Xiaomi Gateway no longer supported

### ISSUE THREAD

**Issue by rvalle** [openhab/openhab-addons#9160]
Hi Community!

I decided to start using OpenHAB and get started with a Xiaomi kit.

Unfortunately the documentation about the Xiaomi gateway is way out of date.

The development mode seems no longer possible to enable and many users have reported it on the forum in the last couple of years.

I warning should be included in the docs so that users either source and old and compatible xiaomi gateway or go for the alternative zigbee to mtqq adapter.

**Comment by Cossey** [comment]
This is what I do to get to developer mode and view the key (or add new sound effects to bridge etc).

I use _BlueStacks Emulator_ to run android on PC and then _APK Mirror_ to download an old version of Mi Home (like 5.5.8) and transfer to the _BlueStacks emulator_ using the _Media Manager_ app in _BlueStacks_. Then you can easily view dev stuff by clicking on bridge, then "..." and then about, and click on the Plugin version heaps until it shows the "Wireless communication protocol" and "Hub info" settings.

Just ignore the "Update Firmware" option if it prompts - sometimes you can click away from the prompt to dismiss, other times you just need to close and reopen the Mi Home app.

Both my v2 gateways have version 1.4.1_170.0158 and 1.4.1_150.0143 on them.

**Comment by rvalle** [comment]
I managed to install an old Mi Home APK on my phone. Wouldn't that test be equivalent to the one you did?

I could not get my V3 gateway to show up in the APP. Some suggest that from V3 you need to register the MI account in the same region where you purchased the gateway. Some say the Gateway needs to be purchased in China to be able to open dev mode.

Perhaps you can contribute to this thread: https://community.openhab.org/t/xiaomi-control-hub-not-compatible-anymore/109104/9 

I still have my gateway here, in case any  test can be performed

**Comment by Cossey** [comment]
Are you referring to the "multimode" gateway like the one below?

![hub](https://user-images.githubusercontent.com/6280250/104911796-4899e380-59f0-11eb-9a2d-7365470cfb50.jpg)

**Comment by rvalle** [comment]
No, mine looks like this one: https://ae01.alicdn.com/kf/HTB1SOyvXyDxK1Rjy1zcq6yGeXXaI/Original-2018-Xiaomi-Mijia-Smart-Home-Multifunctional-Gateway-2-Alarm-System-Intelligent-Online-Radio-Night-Light.jpg

I got it recently in the Mi Shop in Sweden.

**Comment by lolodomo** [comment]
@marcelrv : WDYT ?

**Comment by marcelrv** [comment]
> @marcelrv : WDYT ?

I think this is related to the mihome binding. something I have never deeply looked into.

Note1:  referring to the `developer mode` enabling mentioned above, IF the device supports it, the miio binding can be used to enable the developer mode, no need to find an ancient version of the mihome and install it in bluestacks. 
BUT to my knowledge newer firmware versions just removed that option, hence the difficulties for the mihome binding to connect to these newer ones as it relies on the protocol used for the developer mode.

Note2: there is some very preliminary support for these newer gateways in the miio binding, in the sense that you prob can discover the connected child devices and send commands to them, but this is fully untested except for the one model that I own. Also the responses from these child devices are sometimes bit complex json, so parsing them is bit challenging.

**Comment by lsiepel** [comment]
Older issue so q uick check; do we need to adapt the documentation, remove the mihome binding or what can we do to resolve this?
